engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -300,4 +295,15 @@ public int compare(RenderableChunk o1, RenderableChunk o2) {
         }
     }
 
+	@Override
+	public RenderingConfig getRenderingConfig() {
```

Comment:
Ok, I'm asking around on how to do this: there are people who know injection better. @msteiger? How do you make @In work in a class where it doesn't? We'd like to write something like:

```java
@In 
private WorldProvider worldProvider;
```

but worldProvider is null when you use it, even though it is alive.

Meanwhile, @fabriond, let's not use setters. WorldRendererImpl has the Context passed-in on construction. The Context has references to both WorldProvider and RenderingConfig stored in it. Pass the Context to the camera constructor and you have everything you need.